index.js

```diff
@@ -47,7 +51,7 @@ module.exports = {
         'arrow-body-style': [1, 'as-needed'],
         'prefer-rest-params': 1,
         'no-unneeded-ternary': [1, { defaultAssignment: false }],
-        'arrow-parens': [1, 'as-needed'],
+        'arrow-parens': 0,
```

Comment:
This is sneaky 🗡️

Reply:
It's in the commit message :slightly_smiling_face:

Reply:
Ah, usually comments appear in the desc.

Reply:
Ah yes, I :snake: -ily force pushed without updating the PR description :smile: